cpp/data/system_updates/131.ixtheo.sql

```diff
@@ -0,0 +1 @@
+ALTER TABLE keyword_translations ADD COLUMN priority_entry TINYINT(1) DEFAULT false;
```

Comment:
We could add `AFTER german_updated` to maintain the same order as the installer (so later diffs are easier if we change multiple columns)